app/assets/v2/js/pages/profile-trust.js

```diff
@@ -2246,4 +2246,174 @@ $(document).ready(function() {
+        `Open '${_3BOX_SPACE}' storage space on 3Box`,
+        `Opening '${_3BOX_SPACE}' storage space on 3Box`,
+        `Opened '${_3BOX_SPACE}' storage space on 3Box`
+      ],
+      variable: [
+        `Save passport credential as '${_3BOX_FIELD}'`,
+        `Saving passport credential as '${_3BOX_FIELD}'`,
+        `Saved passport credential as '${_3BOX_FIELD}'`
+      ]
+    };
+
+    const getMessage = (source, state) => {
+      if (state === true) {
+        return '<s>' + source[2] + '</s> ✓';
+      } else if (state === false) {
+        return '<strong>' + source[1] + '</strong>';
+      }
+      return '<em>' + source[0] + '</em>';
+    };
+
+    const get3BoxContent = (auth, storage, variable) => {
+      let list = '<ul>';
+
+      list += '<li>';
+      list += getMessage(messages.auth, auth);
+      list += '</li>';
+
+      list += '<li>';
+      list += getMessage(messages.storage, storage);
+      list += '</li>';
+
+      list += '<li>';
+      list += getMessage(messages.variable, variable);
+      list += '</li>';
+
+      list += '</ul>';
+
+      return `
+        <div>
+          <p><strong>3Box integration:</strong> you can save a POPP credential to 3Box.</p>
+          ${list}
+        </div>
+      `;
+    };
+
+    const vcPassport = () => {
+      if (document.web3network != 'rinkeby' && document.web3network != 'mainnet') {
+        _alert('Please connect your web3 wallet to mainnet + unlock it', 'danger', 1000);
+        return;
+      }
+
+      const accounts = web3.eth.getAccounts();
+
+      $.when(accounts).then((result) => {
+        ethAddress = result[0];
+        ethProvider = web3.currentProvider;
+
+        $.get('/passport/verifiable-credential', params(), async function(response) {
+          let status = response['status'];
+
+          if (status == 'error') {
+            _alert(response['msg'], 'danger', 5000);
+            return;
+          }
+
+          credential = response.vc;
+          verifier = response.verifier;
+
+          $('.modal-body').html(baseContent());
+        });
+      });
+    };
+
+    const vc3Box = async() => {
+      const modal = $('.modal-body');
+      const base = baseContent();
+
+      modal.html(`${base} <hr /> ${get3BoxContent(false)}`);
+
+      const storageBox = await Box.openBox(ethAddress, ethProvider);
+
+      modal.html(`${base} <hr /> ${get3BoxContent(true, false)}`);
+
+      const storageSpace = await storageBox.openSpace(_3BOX_SPACE);
+
+      await storageSpace.syncDone;
+
+      modal.html(`${base} <hr /> ${get3BoxContent(true, true, false)}`);
+
+      const json = JSON.stringify(credential);
+
+      await storageSpace.public.set(_3BOX_FIELD, json);
+
+      modal.html(`${base} <hr /> ${get3BoxContent(true, true, true)}`);
+
+      _alert('Your passport has been uploaded to 3Box', 'success', 5000);
+    };
+
+    const vcCopy = () => {
+      const str = JSON.stringify(credential, null, 2);
+
+      navigator.clipboard.writeText(str);
+    };
```

Comment:
please use the code on clipboard.js 
you can use it like 
<button class="btn btn-primary btn-sm" data-copyclipboard="#shareText">Copy Text</button> 
<div id="shareText"> this string will be copied</div>

Reply:
is not using the last clipboard api but that could be updated if required (we already did it on our clipboard component in vue-components.js

Reply:
Resolved in https://github.com/gitcoinco/web/pull/9032/commits/e32dea61a3a42ebb5281a21695b4eb15cfa6450a.